docs/how-to/rocm-for-hpc/index.rst

```diff
@@ -0,0 +1,239 @@
+.. meta::
+   :description: How to use ROCm for HPC
+   :keywords: ROCm, AI, high performance computing, HPC, usage, tutorial
+
+******************
+Using ROCm for HPC
+******************
+
+The ROCm open-source software stack is optimized to extract the best high
+performance computing (HPC) workload performance from AMD Instinct™ accelerators
+while maintaining compatibility with industry software frameworks.
+
+ROCm enhances support and access for developers by providing streamlined and
+improved tools that significantly increase productivity. Being open-source, ROCm
+fosters innovation, differentiation, and collaboration within the developer
+community, making it a powerful and accessible solution for leveraging the full
+potential of AMD accelerators' capabilities in diverse computational
+applications.
+
+* For more information, see :doc:`What is ROCm? <../../what-is-rocm>`.
+
+* For guidance on installing ROCm, see :doc:`rocm-install-on-linux:index`. See
+  the :doc:`../../compatibility/compatibility-matrix` for details on hardware
+  and operating system support.
+
+Some of the most popular HPC frameworks are part of the ROCm platform, including
+those to help parallelize operations across multiple accelerators and servers,
+handle memory hierarchies, and solve linear systems.
+
+.. image:: ../../data/how-to/rocm-for-hpc/hpc-stack-2024_6_20.png
+   :align: center
+   :alt: Software and hardware ecosystem surrounding ROCm and AMD Instinct for HPC
+
+The following catalog of GPU-accelerated solutions includes a vast set of
```

Comment:
Is it obvious that we have our own version of these applications? 
Does this need to be made more clear?

Have we considered having two links to the applications?
The AMD supported version (already in table) and the main version?

Reply:
> Is it obvious that we have our own version of these applications? Does this need to be made more clear?
> 
> Have we considered having two links to the applications? The AMD supported version (already in table) and the main version?

I wouldn't say we have our own versions. The InfinityHub-CI repo that we link to just provides instructions on building, running, and deploying containerized (with Docker, etc.) HPC solutions w/ ROCm on AMD hardware.